common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4356,11 +4373,12 @@ object CelebornConf extends Logging {
   val DYNAMIC_CONFIG_STORE_BACKEND: ConfigEntry[String] =
     buildConf("celeborn.dynamicConfig.store.backend")
       .categories("master", "worker")
-      .doc("Store backend for dynamic config. Available options: NONE, FS. Note: NONE means disabling dynamic config store.")
+      .doc("Store backend for dynamic config service. Available options: NONE, FS, DB. Note: NONE means disabling " +
```

Comment:
Could we describe mort details for `FS` and `DB` to tell users which options should configure after the `celeborn.dynamicConfig.store.backend` is `FS` or `DB`?